Cargo.toml

```diff
@@ -48,3 +48,6 @@ opt-level = 3
 codegen-units = 1
 opt-level = 'z' # Optimize for size.
 #strip="symbols"
+
+[patch."https://github.com/prisma/quaint"]
```

Comment:
reminder to remove this before merging

Reply:
> Some validation of input values, so fail gracefully if giving negative values from the client.
We can't validate that, the QE doesn't know about native types.